packages/components/src/text/utils.ts

```diff
@@ -71,7 +90,7 @@ export function createHighlighterText( {
 	searchWords = [],
 	unhighlightClassName = '',
 	unhighlightStyle,
-} ) {
+}: Options ) {
```

Comment:
We should remove the `Options` param type from the JSDoc snippet, ie. from `@param {Options} options` to `@param options`

Reply:
Looks like going to:
```ts
/**
 * @param options
 */
 ```

There's an eslint rule that would like to explode it to:
```ts
* @param options.activeClassName
* ...
* /```

Reply:
I guess so? The important part is that we don't duplicate type information in JSDocs, since we have it in TypeScipt now

Reply:
Ack.